### PR TITLE
Hifis: Story Page Hero Section

### PR DESCRIPTION
**Pull request by jean-zhang**
### Note: Chrome dev tools don't work well with the horizontal scrolling for desktop

Only the first 6 commits are relevant, the others are from rebasing (except for the last one)

### CODE REVIEW

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
For this part, could we make this main quote width smaller and also add in the smaller quote that's on the hi-fis? 
image

image

File: components/story-page/story-hero/author-info/AuthorInfo.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    instagram,
+    twitter,
+  } = authorInfo;
+  return (
+    <AuthorInfoWrapper>
+      <DonateLink href={donateLink.url}>
+        <DonateCircularText src={icons.DONATE_CIRCULAR_TEXT} />
+      </DonateLink>
+      <div>
+        <AuthorName>{getString(author_name)}</AuthorName>
```
Comment (isabellberzsenyi) [inline components/story-page/story-hero/author-info/AuthorInfo.js:27]: from hi-fi looks like Author Name should be all caps

File: components/story-page/story-hero/author-info/AuthorInfo.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  const {
+    additional_information,
+    author_name,
+    title,
+    instagram,
+    twitter,
+  } = authorInfo;
+  return (
+    <AuthorInfoWrapper>
+      <DonateLink href={donateLink.url}>
```
Comment (isabellberzsenyi) [inline components/story-page/story-hero/author-info/AuthorInfo.js]: let's add ` target=”_blank”` to all the links (also on line 30 and 33)

**Review by isabellberzsenyi — APPROVED** [review]
B-E-A-UTIFUL

**Pull request merged**